_health-care/_js/components/military-service/ServiceInformationSection.jsx

```diff
@@ -22,9 +22,8 @@ class ServiceInformationSection extends React.Component {
                 onUserInput={(update) => {this.props.onStateChange('lastServiceBranch', update);}}/>
           </div>
           <div className="small-12 columns">
-            <p>
-              Last entry date
-            </p>
+            <label>Last entry date</label>
+            <span className="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
```

Comment:
Can we build the hint into the `DateInput` component since we need it each time that component appears?